src/components/Map/Map.tsx

```diff
@@ -25,6 +25,11 @@ const StyledMap = styled(MapComponent)`
   &.leaflet-drag-target {
     cursor: all-scroll;
   }
+
+  .leaflet-control-attribution a,
+  span {
+    display: none;
+  }
 `
```

Comment:
This wil also hide the map references since all spans are set to display none. If you just select the first span it wil just hide the `|` and leaflet references.


```suggestion
.leaflet-control-attribution {
    a,
    span:first-of-type {
      display: none;
    }
  }
```

Reply:
Even better is not to use CSS but set the prefix of the attributionControl to `false` on the maps that shouldn't show it.

Example: `map.attributionControl.setPrefix(false)`

Reply:
For now I went with your suggestion for the CSS solution, as I think this is the most robust solution.